tests/phpunit/Mutator/Boolean/FalseValueTest.php

```diff
@@ -109,4 +109,15 @@ public function test_does_not_mutate_true_value(): void
 
         $this->assertFalse($this->mutator->canMutate($falseValue));
     }
+
+    public function test_does_not_mutate_switch_stmt(): void
+    {
+        $inputCode = <<<'PHP'
+            <?php
+
+            switch (false) {}
+            PHP;
+
+        $this->assertInputNotMutated($inputCode);
+    }
```

Comment:
actually, you don't need to create separate method to test the case when no mutation is happened.

Please, look here to get an example:

https://github.com/infection/infection/blob/a663d6cecee222e9b33f11608073c134eb5e6916/tests/phpunit/Mutator/Boolean/FalseValueTest.php#L74-L81

so you basically need to create the same `yield` with swith. Thus, new created method `assertInputNotMutated` can also be removed